packages/core/contracts/data-verification-mechanism/implementation/FixedSlashSlashingLibrary.sol

```diff
@@ -76,16 +76,16 @@ contract FixedSlashSlashingLibrary is SlashingLibraryInterface {
      * @param totalCorrectVotes The total amount of correct votes.
      * @param priceRequestIndex The price request index within the resolvedPriceRequestIds array.
      * @param isGovernance Whether the request is a governance request.
-     * @return wrongVoteSlashPerToken The amount of tokens to slash for voting wrong.
-     * @return noVoteSlashPerToken The amount of tokens to slash for not voting.
+     * @return uint256 The amount of tokens to slash for voting wrong.
+     * @return uint256 The amount of tokens to slash for not voting.
      */
     function calcSlashing(
         uint256 totalStaked,
         uint256 totalVotes,
         uint256 totalCorrectVotes,
         uint256 priceRequestIndex,
         bool isGovernance
-    ) external view returns (uint256 wrongVoteSlashPerToken, uint256 noVoteSlashPerToken) {
```

Comment:
so I dont think we want to do this in some cases. This will break other parts of how we use these props, in particular I think this will break this code here:
https://github.com/UMAprotocol/protocol/blob/923f378db029f272eba1317296d9aa0c395c64fd/packages/core/contracts/data-verification-mechanism/implementation/VotingV2.sol#L843

where we reference .`wrongVoteSlashPerToken` and `noVoteSlashPerToken` from the return type. without this we'd need to treat this as an array.

The feedback we had last time OZ complained about this was that our normal rule is that: a) in all cases we don't define returned name props except for b) if there is more than one returned prop, then we return named props. This does diverge from the normal spec but it helps a lot with code cleanness. 

I think you'll also see that a number of the tests have now broken as a result of this.

Reply:
agree, then I believe there is nothing we should do on N-04 and N-05